math/libfaiss/Portfile

```diff
@@ -4,9 +4,10 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        facebookresearch faiss 1.6.4 v
+github.setup        facebookresearch faiss 1.7.0 v
 name                libfaiss
 epoch               1
+revision            0
```

Comment:
If the revision is not set it is implicitly 0, so this line is unnecessary.

Reply:
Unnecessary but convenient to keep around? Lots of ports set `revision 0`, and it's a nice reminder in the workflow. Personally, I prefer it there.